tests/files/drupal8-mariadb-single/web/sites/default/default.settings.php

```diff
@@ -119,7 +119,7 @@
  * That is useful for primary/replica replication, as Drupal may try to connect
  * to a replica server when appropriate and if one is not available will simply
  * fall back to the single primary server (The terms primary/replica are
```

Comment:
I think we can remove that parenthetical now :)

Reply:
It was already there, I only replaced the `master/slave` instances with `primary/replica` - I _think_ these files are supposed to be the default Drupal provided settings files